### PR TITLE
Editor HDR Color Grading Component implemented under the PostFx stack

### PR DESCRIPTION
**Pull request by hershey5045**
Notes:
- HDR Color Grading component implemented under PostFx stack.
- Able to blend with other HDR Color Grading components.
- Added HDR color grading pass.

Points for focus:
- Editor HDR Color Grading Component property widget, description, and tool-tip.
- Grouping of properties.
- Type of color grading variable in HDRColorGradingParams.inl
image

### CODE REVIEW

**Review by HogJonny-AMZN — COMMENTED** [review]
I've looked this over but will leave the code review approval to other SDEs.
After I build this and check out the component I'll add any other comments I have.

File: Gems/Atom/Feature/Common/Assets/ShaderLib/3rdParty/Features/PostProcessing/KelvinToRGB.azsli [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    RGB.xyz          = max( RGB.xyz, 0.000001f );
+    float3 HCV       = RGBToHCV(RGB);
+    float L          = HCV.z - HCV.y * 0.5f;
+    float S          = HCV.y / ( 1.0f - abs( L * 2.0f - 1.0f ) + 0.000001f);
+    return float3( HCV.x, S, L );
+}
+
+float3 HSLToRGB( float3 HSL )
+{
+    float3 RGB       = HUEToRGB(HSL.x);
```
Comment (Adi-Amazon) [inline Gems/Atom/Feature/Common/Assets/ShaderLib/3rdParty/Features/PostProcessing/KelvinToRGB.azsli:47]: not sure why you introduced so many white spaces in the shaders.  I suggest doing a pass and leaving only a single space or short align as used in most of the code.  It is also more readable.

Comment (invertednormal) [reply Gems/Atom/Feature/Common/Assets/ShaderLib/3rdParty/Features/PostProcessing/KelvinToRGB.azsli:47]: I think this is 3rd Party code so it's probably ok to leave in its original form.

Comment (hershey5045) [reply Gems/Atom/Feature/Common/Assets/ShaderLib/3rdParty/Features/PostProcessing/KelvinToRGB.azsli:47]: Sorry this file should not be here, let remove that. (This was committed with the right format on a previous PR)

**Pull request merged**